tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1076,3 +1076,48 @@ Feature: check share activity
         }
       }
       """
+
+  @issue-9849
+  Scenario: sharee tries to check activities of folder using share mountpoint id
+    Given user "Alice" has created folder "/FOLDER"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | /FOLDER  |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    And user "Brian" has a share "/FOLDER" synced
+    And user "Brian" has uploaded file with content "some data" to "Shares/FOLDER/newfile.txt"
+    And user "Brian" has uploaded file with content "edited data" to "Shares/FOLDER/newfile.txt"
+    And user "Brian" has deleted file "Shares/FOLDER/newfile.txt"
+    When user "Brian" lists the activities for folder "FOLDER" of space "Shares" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["value"],
+        "properties": {
+          "value": {
+            "type": "array",
+            "minItems": 0,
+            "maxItems": 0,
+            "uniqueItems": true,
+            "items": {
+            }
+          }
+        }
+      }
+      """
+
+  @issue-9849
+  Scenario: sharee tries to check activities of another unshared file
+    Given user "Alice" has uploaded file with content "another ownCloud test text file" to "anotherTextfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Brian" tries to list the activities of unshared file "anotherTextfile.txt" of user "Alice" from space "Personal" using the Graph API
```

Comment:
```suggestion
    When user "Brian" tries to list the activities of file "anotherTextfile.txt" owned by user "Alice" using the Graph API
```

Reply:
@saw-jan I think we need the spaceName as well to get the resourceId in this step. What do you think?

Reply:
ahh, then we can accommodate space name as well
`When user "Brian" tries to list the activities of file "anotherTextfile.txt" from space "Personal" owned by user "Alice" using the Graph API`